wallet/src/blockchain.rs

```diff
@@ -33,4 +39,18 @@ impl Settings {
 
         value > minimal_value
     }
+
+    // extract the block0 date-time
+    // seconds in unix time (seconds elapsed since 1-Jan-1970)
+    pub fn start_date_time(&self) -> SystemTime {
+        SystemTime::UNIX_EPOCH + Duration::from_secs(self.static_parameters.block0_start_time.0)
+    }
+
+    pub fn discrimination(&self) -> Discrimination {
+        self.static_parameters.discrimination
+    }
+
+    pub fn time_era(&self) -> &TimeEra {
```

Comment:
I'm not sure if with this is enough, or should I make some methods to explicitly expose the data inside?

Reply:
it's more than enough for now, having them available around here allows us to know what we can do for the next steps.